src/App.vue

```diff
@@ -11,41 +11,73 @@
       overflow
       app
     />
-    <v-toolbar :clipped-left="primaryDrawer.clipped" app absolute>
+    <v-toolbar
+      :clipped-left="primaryDrawer.clipped"
+      app
+      absolute
+    >
       <v-toolbar-side-icon
         v-if="primaryDrawer.type !== 'permanent'"
         @click.stop="primaryDrawer.model = !primaryDrawer.model"
       />
       <v-toolbar-title class="headline text-uppercase">
         <span class="font-weight-light">{{ app_name }}</span>
       </v-toolbar-title>
-      <v-spacer/>
-      <v-btn to="/" flat>Home</v-btn>
-      <v-btn to="/about" flat>About</v-btn>
-      <v-btn to="/dev" flat>Dev</v-btn>
-      <v-btn to="/maps" flat>Maps</v-btn>
-      <v-btn to="/settings" flat>Settings</v-btn>
+      <v-spacer />
+      <v-btn
```

Comment:
Use this style of writing attributes (I'm about line breaks) only with long attributes And number of them > 2